src/appleseed/foundation/utility/bufferedfile.h

```diff
@@ -203,6 +205,29 @@ class APPLESEED_DLLSYMBOL BufferedFile
 };
 
 
+//
+// Utility free functions.
+//
+
+struct ExceptionEOF : public Exception {};
```

Comment:
It would have made sense to. So far the approach is to promote exception types to `foundation/core/exceptions/` only when they're commonly used, in multiple places.